spot/reentrancy-demo.ipynb

```diff
@@ -0,0 +1,830 @@
+      "\u001b[0m\u001b[K\u001b[0m\u001b[?25h\u001b[0m✨ Mounting… done\n",
+      "\u001b[0m✨ \u001b[0mRun is running\n",
+      "\u001b[0mFinished epoch 0.\n",
+      "\u001b[0mSaved model to 0_net.pth.\n",
+      "\u001b[0mFinished epoch 1.\n",
+      "\u001b[0mFinished epoch 2.\n",
+      "\u001b[0mFinished epoch 3.\n",
+      "\u001b[0mFinished epoch 4.\n",
+      "\u001b[0mFinished epoch 5.\n",
+      "\u001b[0mSaved model to 5_net.pth.\n",
+      "\u001b[0mFinished epoch 6.\n",
+      "\u001b[0mFinished epoch 7.\n",
+      "\u001b[0mFinished epoch 8.\n",
+      "\u001b[0mFinished epoch 9.\n",
+      "\u001b[0mFinished epoch 10.\n",
+      "\u001b[0mSaved model to 10_net.pth.\n",
+      "\u001b[0mFinished epoch 11.\n",
+      "\u001b[0mFinished epoch 12.\n",
+      "\u001b[0mFinished epoch 13.\n",
+      "\u001b[0mFinished epoch 14.\n",
+      "\u001b[0mFinished epoch 15.\n",
+      "\u001b[0mSaved model to 15_net.pth.\n",
+      "\u001b[0mFinished epoch 16.\n",
+      "\u001b[0mFinished epoch 17.\n",
+      "\u001b[0mFinished epoch 18.\n",
+      "\u001b[0mFinished epoch 19.\n",
+      "\u001b[0mFinished epoch 20.\n",
+      "\u001b[0mSaved model to 20_net.pth.\n",
+      "\u001b[0mFinished epoch 21.\n",
+      "\u001b[0mFinished epoch 22.\n",
+      "\u001b[0mFinished epoch 23.\n",
+      "\u001b[0mFinished epoch 24.\n",
+      "\u001b[0mFinished epoch 25.\n",
+      "\u001b[0mSaved model to 25_net.pth.\n",
+      "\u001b[0mFinished epoch 26.\n",
+      "\u001b[0mFinished epoch 27.\n",
+      "\u001b[0mFinished epoch 28.\n",
+      "\u001b[0mFinished epoch 29.\n",
+      "\u001b[0mFinished epoch 30.\n",
+      "\u001b[0mSaved model to 30_net.pth.\n",
+      "\u001b[0mFinished epoch 31.\n",
+      "\u001b[0mFinished epoch 32.\n",
+      "\u001b[0mFinished epoch 33.\n",
+      "\u001b[0mFinished epoch 34.\n",
+      "\u001b[0mFinished epoch 35.\n",
+      "\u001b[0mSaved model to 35_net.pth.\n",
+      "\u001b[0m^C\n",
+      "\n",
+      "\u001b[0m✨ Your run is still running remotely.\n",
+      "\u001b[0m✨ Use 'spell kill 307' to terminate your run\n",
+      "\u001b[0m✨ Use 'spell logs 307' to view logs again\n",
+      "\u001b[0m\u001b[K\u001b[0m\u001b[?25h\u001b[0m\u001b[0m"
+     ]
+    }
+   ],
+   "source": [
+    "!spell run --machine-type V100 \\\n",
+    "    --github-url 'https://github.com/spellrun/spell-examples.git' \\\n",
+    "    --github-ref reentrancy \\\n",
+    "    --mount uploads/segmented-bob-ross-images:/mnt/segmented-bob-ross-images \\\n",
+    "    --pip Pillow \\\n",
+    "    \"python spot/train_reentrant.py\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Spell workspace will not log you in by default. If you are running this notebook from inside of a Spell workspace you will need to run the following command, replacing `YOUR_EMAIL` with your Spell email and `YOUR_PASSWORD` with your Spell password:"
```

Comment:
it should log you in now!

Reply:
Didn't work for me. We should discuss in follow-up.